### ISSUE TITLE
How to include multiple files into the diagram?

### ISSUE THREAD

**Issue by mostley** [myxvisual/vscode-ts-uml#3]
As far as I can see there's no way to add multiple files yet, is there?
It looks like the Diagram would support it since it adds a Window for a file. It would be great if it could show more that AliasExcludes for a dependency.

Apart from that very nicely done!

**Comment by zhusjfaker** [comment]
same to me 

i want to  include multiple files into the diagram

When can this feature be supported

**Comment by rubengmurray** [comment]
Agreed. Is this being openly maintained? Would a PR that attempts to add in this functionality be welcomed?

I'm dying to find a diagram generator in vs code that incorporates multiple files, that looks relatively modern, and more to the point, incorporates the interactions between the files. @myxvisual

**Comment by miladfm** [comment]
Is there any update on this Feature?

**Comment by stealthrabbi** [comment]
Yeah, a UML diagram with a single class isn't particularly useful. Aren't you supposed to have differnt files for each type? Right-clicking a folder and selecting "Show Typescript UML from folder" generates a blank diagram.
